lib/pnr.test.js

```diff
@@ -69,6 +69,11 @@ describe(`regex`, () => {
   test(`Should match pnrs with a trailing character`, () => {
     expect(Pnr.regex.test(`198511179882L`)).toBeTruthy();
   });
+  test(`Should not match if pnr is inside numberserie`, () => {
```

Comment:
I do not like it when there are two runs of the method under test in the same test case, in this case `Pnr.regex.test`. Maybe split it into one test for "should not match if pnr is inside phone number" and one should not match if pnr is inside serialVersionId"?

Reply:
Fixed that, even if I don't totally agree, there are a lot of testcases with multiple asserts, and these is  testing the same thing, i just took the cases from the raised issue.  But anyway, I have changed it.

Reply:
Great! It's not multiple asserts that I see as a problem, what I see as a problem is multiple runs of the method under test in one test case.

This is fine:
```
const result = myMethod()
expect(result.length).toBe(2);
expect(result.size).toBe(2);
```

This I do not like:
```
expect(myMethod().length).toBe(2);
expect(myMethod().size).toBe(2);
```